channels/jaicp/src/main/kotlin/com/justai/jaicf/channel/jaicp/reactions/JaicpReactions.kt

```diff
@@ -20,11 +24,40 @@ open class JaicpReactions : Reactions() {
         return SayReaction.create(text)
     }
 
-    fun collect(): JsonObject {
-        val jsonReplies = replies.map { reply ->
-            reply.serialized().toJson()
-        }
+    /**
+     * Starts a new conversation session with client.
+     *
+     * Each session is logged to JAICP analytics with unique session identifier.
+     * Using this reaction creates new session identifier, clears existing [DialogContext]
+     *  and sends a response with new session identifier.
+     *
+     * @see [com.justai.jaicf.channel.jaicp.logging.JaicpConversationLogger]
+     * */
+    fun startNewSession() {
+        botContext.dialogContext.currentState = "/"
```

Comment:
I guess it's better to use similar to JAICP naming scheme for JAICP-channels. 
And you're right, it should not reset `DialogContext`. This will be fixed in further commit.